dex/order/match.go

```diff
@@ -76,6 +76,7 @@ const (
 	// their redemption transaction. The DEX has validated the redemption and
 	// sent the details to the maker.
 	MatchComplete // 4
+	//MatchRefunded // 5?
```

Comment:
```go
//MatchRevoked // 6?
```

Reply:
Only thing with a revoked status is that a revoked match that first got at least to the actor broadcasting swap should eventually end up as MatchRefunded or MatchRedeemed (with the auto-redeem that's in the works).